blocks/full-author-bio-block/features/full-author-bio/_children/Presentation/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+
+import getTranslatedPhrases from "fusion:intl";
+
+import {
+	formatSocialURL,
+	Heading,
+	HeadingSection,
+	Icon,
+	Image,
+	Link,
+	MediaItem,
+	Paragraph,
+	Stack,
+} from "@wpmedia/arc-themes-components";
+
+const socialIcons = {
+	email: "Envelope",
+	facebook: "Facebook",
+	instagram: "Instagram",
+	linkedin: "LinkedIn",
+	medium: "Medium",
+	pinterest: "Pinterest",
+	reddit: "Reddit",
+	rss: "Rss",
+	snapchat: "Snapchat",
+	soundcloud: "SoundCloud",
+	tumblr: "Tumblr",
+	twitter: "Twitter",
+	whatsapp: "WhatsApp",
+	youtube: "Youtube",
+};
+
+const BLOCK_CLASS_NAME = "b-full-author-bio";
+
+const Presentation = ({ author = {}, locale }) => {
+	const phrases = getTranslatedPhrases(locale);
+	const supportedSocials = Object.keys(socialIcons);
+	const socials = Object.entries(author)
+		.filter(([key, value]) => supportedSocials.includes(key) && value)
+		.map(([key]) => key);
+
+	const bio = author?.bio || author?.longBio;
+
+	return author?.byline || author?.image || author?.role || bio || socials.length ? (
+		<div className={BLOCK_CLASS_NAME}>
+			{author?.image ? (
+				<MediaItem>
+					<Image src={author.image} alt={author.byline || ""} />
+				</MediaItem>
+			) : null}
+			{author?.byline || author?.role || bio || socials.length ? (
+				<HeadingSection>
+					<Stack className={`${BLOCK_CLASS_NAME}__text`}>
+						{author?.byline || author?.role ? (
+							<Stack className={`${BLOCK_CLASS_NAME}__identification`}>
+								{author.byline ? (
+									<Heading className={`${BLOCK_CLASS_NAME}__name`}>{author.byline}</Heading>
+								) : null}
+								{author.role ? (
+									<HeadingSection>
+										<Heading className={`${BLOCK_CLASS_NAME}__role`}>{author.role}</Heading>
+									</HeadingSection>
+								) : null}
+							</Stack>
+						) : null}
+						{bio ? <Paragraph>{bio}</Paragraph> : null}
+						{socials.length ? (
+							<Stack className={`${BLOCK_CLASS_NAME}__social`}>
+								<HeadingSection>
+									<Heading className={`${BLOCK_CLASS_NAME}__social-header`}>
+										{phrases.t("full-author-bio-block.connect-text")}
+									</Heading>
+								</HeadingSection>
+								<div className={`${BLOCK_CLASS_NAME}__social-icons`}>
+									{socials.map((item) => (
+										<Link
+											className={`${BLOCK_CLASS_NAME}__social-link`}
+											href={formatSocialURL(item, author[item])}
+											key={item}
+											openInNewTab
+											title={phrases.t(`global.social-${item.toLowerCase()}-content`, {
```

Comment:
These phrases are not found with the changes in this pull requests - does this pull request depend on #1474 ?

Reply:
Yeah:
![image](https://user-images.githubusercontent.com/2287238/186943539-0151a731-6733-45a7-8ee6-d996e2326d36.png)